src/main/scala/com.snowplowanalytics.snowplow.micro/ConfigHelper.scala

```diff
@@ -81,32 +84,42 @@ private[micro] object ConfigHelper {
             case None => success
           }
         )
+      opt[URI]("extra-iglu-resolver")
```

Comment:
To borrow [the terminology from our documentation site](https://docs.snowplow.io/docs/pipeline-components-and-applications/iglu/), I believe this is config parameter describes a "registry" not a "resolver".  A registry is the thing that hosts schemas.  A resolver is the thing that tells the client which registries to use.